crates/fluvio-cli/src/client/hub/smartmodule/list.rs

```diff
@@ -91,12 +108,20 @@ mod output {
 
         /// table content implementation
         fn content(&self) -> Vec<Row> {
-            self.0
+            self.pkgs
                 .iter()
                 .map(|e| {
                     Row::from([
                         Cell::new(e.pkg_name()).set_alignment(CellAlignment::Left),
                         Cell::new(&e.visibility).set_alignment(CellAlignment::Left),
+                        Cell::new(
+                            e.published_at()
+                                .map(|date| {
+                                    TimeElapsedFormatter::default()
+                                        .time_since_str(date, self.exact_time)
```

Comment:
prefer human time as default since that's how typically most tools shows

Reply:
Yep! `exact_time` is default to `false` in CLI's

https://github.com/infinyon/fluvio/blob/ba6348ad15e02909fbd3aa381f87f2df0ce26413/crates/fluvio-cli/src/client/hub/smartmodule/list.rs#L19-L21

https://github.com/infinyon/fluvio/blob/ba6348ad15e02909fbd3aa381f87f2df0ce26413/crates/fluvio-hub-util/src/cmd/list.rs#L19-L21